dist/profile/templates/confluence/vhost.conf

```
@@ -17,6 +17,8 @@ RewriteRule ^.* - [F,L]
 
 # Rewrite all plugin docs to plugins.jenkins.io if user agent is not the wiki exporter
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Writinng\+an\+SCM\+plugin$" "https://plugins.jenkins.io/doc/developer/plugin-development/writing-an-scm-plugin" [NC,L,QSA,R=301]
```

Sorry I missed the actual error on my first review.  The redirect needs to go to the www.jenkins.io page not the plugins.jenkins.io page.
```suggestion
RewriteRule "^/display/JENKINS/Writinng\+an\+SCM\+plugin$" "https://www.jenkins.io/doc/developer/plugin-development/writing-an-scm-plugin" [NC,L,QSA,R=301]
```